service_storagefacility/app/controllers/StorageController.scala

```diff
@@ -484,8 +484,8 @@ final class StorageController @Inject()(
   def search(
       mid: Int,
       searchStr: Option[String],
-      page: Int = 1,
```

Comment:
I would double check your statement here. The line you comment was modified to have the default value explicitly stated in the routes file instead of both places (as it used to have) ;-)